tests/foreman/api/test_product.py

```diff
@@ -1,52 +0,0 @@
-# -*- encoding: utf-8 -*-
```

Comment:
I never did figure out how to use the old API tests. I have a very, very hard time understand how they work, much less why any failures occur.